### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by dgrassellyb**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately 
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@dgrassellyb thanks for this! do you want to add a brief 1 liner to the NEWS.md file for this and the other PR earlier updating the default CI workflows? its a bit of a nice to have for something like this template repo as people would always just take the latest, but can't harm to call out what has changed over time.

**Comment by dgrassellyb** [comment]
> @dgrassellyb thanks for this! do you want to add a brief 1 liner to the NEWS.md file for this and the other PR earlier updating the default CI workflows? its a bit of a nice to have for something like this template repo as people would always just take the latest, but can't harm to call out what has changed over time.

Sure ! I guess we can stay bit high level (the main change actually concerns the fact we get rid of `renv` + introducing new docker images with new version of R !)

**Comment by rossfarrugia** [comment]
agree @dgrassellyb - if you can add something brief then i'll approve and merge.

**Comment by dgrassellyb** [comment]
> agree @dgrassellyb - if you can add something brief then i'll approve and merge.

done ! I also deleted renv releated conf because we're not using `renv` anymore !

**Review by rossfarrugia — APPROVED** [review]
thanks @dgrassellyb!

@bms63 @manciniedoardo do you want to sanity check anything here before we merge?

File: NEWS.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
  - Package Extension guidance on the front page was updated and moved to [this location](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html) in `{admiraldev}`, and a link was added to point to the new location (#61).
```
Comment (manciniedoardo) [inline NEWS.md]: This link doesn't work for me, but [this one does].

**Review by manciniedoardo — COMMENTED** [review]
Just a note about a broken link.

File: NEWS.md (hunk 1 above)
Comment (dgrassellyb) [reply NEWS.md]: I don't know much about what you want to refer from admiraldev here exactly, maybe we can just copy paste what we have from admiral readme ?

Comment (dgrassellyb) [reply NEWS.md]: (@rossfarrugia / @manciniedoardo : also just fixed the templates - we were using old `admiral.test` => I updated news.md as well on this

Comment (rossfarrugia) [reply NEWS.md]: good spot for admiral.test! i think the admiraldev reference is fine for now

**Review by rossfarrugia — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (rossfarrugia) [reply NEWS.md]: oh sorry missed the broken link comment - @dgrassellyb you just need to remove the `main/` bit in that link and it works then

**Comment by bms63** [comment]
I was thinking on the vbump stuff - do we want to use it in a template package.  This is never going to be released. - we are always going to be in dev mode.

Also I removed the renv stuff in another PR

File: NEWS.md (hunk 1 above)
Comment (dgrassellyb) [reply NEWS.md]: I fixed them on `readme.Rmd` but we will see the updates on `readme.Rmd` only when we will merge to main branch

Comment (rossfarrugia) [reply NEWS.md]: @bms63 @manciniedoardo didn't we get rid of the `README.Rmd` approach now and we only use `README.md`? sorry, every boulder we turn over seems to find a new surprise here, as we've updated so much in admiral conventions since the last time we made a new pkg ext.

Comment (bms63) [reply NEWS.md]: admiral never had a `.Rmd` it was adopted into the other extension packages and admiraldev right about when I came onbaord.  Honestly, I'd like to get rid of them all - I think we had them for helping with dynamic linking but can't remember anymore??

Anyways, I think we should remove here and remove in `peds` as well and maybe make a push for removing them in all packages??

 Try to keep as simple as possible!

Comment (dgrassellyb) [reply NEWS.md]: just deleted it, I think as well it's a bit overkill honestly - for `vbump` I commented it to not use it for `admiraltemplates` directly, but still keep it because some other projects might want to use it

Comment (rossfarrugia) [reply NEWS.md]: i just removed the readme rendering from admiralpeds too

**Comment by bms63** [comment]
I moved the DESCRIPTION and news back to 0.0.3.  Let's merge this in and make other updates as needed.  

Exciting to have the template caught up to current standards.  Nothing like a deadline to make frantic last minute updates!! :)
